src/avfoundation.cs

```diff
@@ -12652,5 +12811,51 @@ interface AVCapturePhoto
 		[iOS (11, 0)]
 		[NullAllowed, Export ("previewCGImageRepresentation")]
 		CGImage PreviewCGImageRepresentation { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (12, 0)]
+		[NullAllowed, Export ("portraitEffectsMatte")]
+		AVPortraitEffectsMatte PortraitEffectsMatte { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (12,0)]
+		[Export ("fileDataRepresentationWithCustomizer:")]
+		[return: NullAllowed]
+		NSData GetFileDataRepresentation (IAVCapturePhotoFileDataRepresentationCustomizer customizer);
+	}
+
+	[NoWatch, TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVPortraitEffectsMatte
+	{
+		[Static]
+		[Export ("portraitEffectsMatteFromDictionaryRepresentation:error:")]
+		[return: NullAllowed]
+		AVPortraitEffectsMatte Create (NSDictionary imageSourceAuxDataInfoDictionary, [NullAllowed] out NSError outError);
+
+		[Export ("portraitEffectsMatteByApplyingExifOrientation:")]
+		AVPortraitEffectsMatte Create (CGImagePropertyOrientation exifOrientation);
+
+		[Export ("portraitEffectsMatteByReplacingPortraitEffectsMatteWithPixelBuffer:error:")]
+		[return: NullAllowed]
+		AVPortraitEffectsMatte Create (CVPixelBuffer pixelBuffer, [NullAllowed] out NSError outError);
+
+		[Export ("dictionaryRepresentationForAuxiliaryDataType:")]
+		[return: NullAllowed]
+		NSDictionary DictionaryRepresentationForAuxiliaryDataType ([NullAllowed] out string outAuxDataType);
```

Comment:
no verb/action, maybe `GetDictionaryRepresentation` ? as the suffix is already implied with the parameter name